test-app/tests/integration/components/headless-form-test.gts

```diff
@@ -230,12 +184,73 @@ module('Integration Component headless-form', function (hooks) {
         assert.dom('input').hasAttribute('type', type, `supports type=${type}`);
```

Comment:
Should we assert that the type here is `text`?

Reply:
We are inside a loop here that asserts that we have a `type` of _all_ supported types, not just `text` (but including it). Or did I misunderstand you?

Reply:
Nope my bad seems like I misread the code.